### DISCUSSION TITLE
Expo SDK 50 and Moti Support

### DISCUSSION THREAD

**Discussion by markmccoid** [nandorojo/moti#335] [Q&A]
I upgraded a project from SDK 49 to SDK 50 and am getting errors like:

"Error: You attempted to set the key `width` with the value `0` on an object that is meant to be immutable and has been frozen."
 
Here is the code that causes this:
```
<MotiView  from={{ opacity: 0 }} animate={{ opacity: 1 }}>
      <Text>Testing</Text>
</MotiView>
```

Any thoughts on something I might be missing?

Thanks!

**Comment by nandorojo** [comment]
Hmm weird, we haven’t seen this since upgrading. Do
You have a repro? Can you try reanimated nightly version?

**Comment by markmccoid** [comment]
I've been playing around a bit more and I think it might be some other issue that is making it look like Moti.  I'm using Nativewind and upgraded to their v4 (still unreleased) and that might be part of the problem.  Going to start the upgrade over.

Thanks for the great library!!
